internal/setup/setup.go

```diff
@@ -596,6 +629,13 @@ func parsePackage(baseDir, pkgName, pkgPath string, data []byte) (*Package, erro
 						info = ""
 					}
 				}
+				generate = yamlPath.Generate
+				if len(yamlPath.Generate) > 0 {
+					kinds = append(kinds, GeneratePath)
+					if err := validateGeneratePath(contPath, generate); err != nil {
+						return nil, fmt.Errorf("slice %s_%s %w", pkgName, sliceName, err)
+					}
+				}
```

Comment:
Now that I think about it, this block can be moved upwards and then we avoid checking `yamlPath.Generate` twice.

Reply:
Oka, only leaving the following line in this block to add whatever values `generate` may have.

```go
				generate = yamlPath.Generate
```